src/components/Schedule/ReservationInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from "react";
+
+interface IProps {
+  time: string;
+  therapyName: string;
+  doctorName: string;
+  place: string;
+  therapyLength: number;
+  price: number;
+}
+
+export const ReservationInfo: FC<IProps> = ({
+  time,
+  therapyName,
+  doctorName,
+  place,
+  therapyLength,
+  price
+}) => (
+  <div className="pt-4">
+    <div className="flex">
+      <div className="text-xl mt-1 font-gilbert">{time}</div>
+      <div
```

Comment:
don't create a div just for border, solve it by adding padding and border to previous text div